lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -443,6 +447,22 @@ defmodule ExUnit.Callbacks do
 
   defp start_supervised_error(reason), do: Exception.format_exit({:start_spec, reason})
 
+  @doc """
+  Same as `start_link_supervised/2` but returns the PID on success and raises if
+  not started properly.
```

Comment:
The docs need to be updated now as the non-bang variant no longer exists. :)

Reply:
Right!